main/main.py

```diff
@@ -58,12 +58,13 @@ def main():
     rr = ReleaseRequest(organisation, project, build_number)
     burgr = Burgr(burgrx_url, burgrx_user, burgrx_password, rr)
 
-    try:
-        burgr.releasability_checks(version, github.current_branch())
-    except Exception as e:
-        print(f"::error releasability did not complete correctly. " + str(e))
-        github.revoke_release()
-        sys.exit(1)
+    # TODO JC ITOPS-3235 commented out for TESTING PURPOSE ONLY
```

Comment:
fix before merging

Reply:
It works if using a `branch-` branch name prefix: https://github.com/SonarSource/sonar-dummy/runs/5273333426?check_suite_focus=true